lib/IconButton/IconButton.js

```diff
@@ -195,5 +210,16 @@ module.exports = kind(
 		if (inEvent.originator === this) {
 			this.bubble('onRequestScrollIntoView');
 		}
+	},
+
+	/**
+	* @private
+	*/
+	backgroundOpacityChanged: function (old) {
```

Comment:
Aaron, could you please tell me  what is the potential gain?? 
I see your point about complicated and needed to be abstracted(less mixin)
The reason why I am asking is this backgroundOpacity may be applied to other controls like videoplayer control(play , pause button....)
Thanks